tck/common/pom.xml

```diff
@@ -24,6 +24,7 @@
         <groupId>jakarta.tck</groupId>
         <artifactId>persistence-tck</artifactId>
         <version>3.2.1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
```

Comment:
But `../pom.xml` is default, so why this is needed?

Reply:
I didn't contact JetBrains about this error, no, because it was a weird permission error. Anyway, I removed these changes for the time being.